thread/linux/omrthreadnuma.c

```diff
@@ -23,16 +23,23 @@
  */
 /* _GNU_SOURCE must be defined for CPU_SETSIZE */
 #define _GNU_SOURCE
+#include <ctype.h>
 #include <dirent.h>
+#include <fcntl.h>
+#include <inttypes.h>
 #include <sched.h>
+#include <sys/stat.h>
 #include <stdio.h>
 
 #include "omrcfg.h"
 #include "threaddef.h"
 
 static uintptr_t cpuset_subset_or_equal(cpu_set_t *subset, cpu_set_t *superset);
 static void cpuset_logical_or(cpu_set_t *destination, const cpu_set_t *source);
-
+#undef OMR_VERBOSE_INIT
```

Comment:
I might pick a more unique name for this.  Something like OMR_VERBOSE_NUMA_NODE_DATA?